spec/lib/tabular_form_builder_spec.rb

```diff
@@ -533,4 +533,93 @@
       end
     end
   end
+
+  # test the label that is generated for various field types
+  describe 'labels for fields' do
+    let(:options) { {} }
+    shared_examples_for "generated label" do
+      def remove_form_field_container(string)
+        string.gsub(/<span class="form--field-container">.+<\/span>/m,'')
+      end
+
+      def expected_label_like(expected_title)
+        expect(remove_form_field_container(output)).to be_html_eql(%{
+          <label class="form--label"
+                 for="user_name"
+                 title="#{expected_title}">
+            #{expected_title}
+          </label>
+        })
+      end
+
+      context 'with a label specified as string' do
+        let(:text) { "My own label" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.